app/src/main/java/com/wootric/androidsdk_demo/MainActivity.java

```diff
@@ -3,31 +3,32 @@
 import android.app.Activity;
 import android.os.Bundle;
 import android.view.View;
+import android.view.Window;
 
 import com.wootric.androidsdk.Wootric;
 
 public class MainActivity extends Activity {
 
-    private static final String CLIENT_ID = "CLIENT ID";
-    private static final String CLIENT_SECRET = "CLIENT SECRET";
-    private static final String ACCOUNT_TOKEN = "ACCOUNT TOKEN";
-
+    private static final String CLIENT_ID = "627ff892837d5762946f4699529642e8cb9833cbaec693f3f9b9f42904c7fd9b";
+    private static final String CLIENT_SECRET = "79a0fdf8ffc56dcc498826f948204ac3506d028d12938fcec49ef2b8db1615ee";
+    private static final String ACCOUNT_TOKEN = "NPS-7768d9aa";
```

Comment:
Please go back to the previous version (without your client id, secret and account token)